Client/Frontend/Shields/ShieldsView.swift

```diff
@@ -38,31 +39,35 @@ extension ShieldsViewController {
         override init(frame: CGRect) {
             super.init(frame: frame)
             
-            addSubview(scrollView)
-            scrollView.addSubview(stackView)
-            
-            scrollView.snp.makeConstraints {
-                $0.edges.equalTo(self)
-            }
-            
-            scrollView.contentLayoutGuide.snp.makeConstraints {
-                $0.width.equalTo(self)
-            }
-            
-            stackView.snp.makeConstraints {
-                $0.edges.equalTo(scrollView.contentLayoutGuide)
-            }
-            
             stackView.addArrangedSubview(shieldOverrideControl)
             stackView.addArrangedSubview(overviewStackView)
             stackView.addArrangedSubview(shieldsContainerStackView)
+            
+            addSubview(scrollView)
+            scrollView.addSubview(stackView)
         }
         
         @available(*, unavailable)
         required init?(coder aDecoder: NSCoder) {
             fatalError()
         }
         
+        override func layoutSubviews() {
+            super.layoutSubviews()
+            
+            scrollView.snp.remakeConstraints {
```

Comment:
Hmm, is there a reason we constrain the scrollView in `layoutSubviews`?
Just wondering if this will trigger another layout.